packages/console/src/components/Checkbox/index.module.scss

```diff
@@ -0,0 +1,45 @@
+@use '@/scss/underscore' as _;
+
+.checkbox {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+
+  .icon {
+    width: 20px;
+    height: 20px;
+    margin-right: _.unit(2);
+    display: none;
+  }
+
+  input {
+    position: absolute;
+    width: 20px;
+    height: 20px;
+    left: 0;
+    top: 0;
+    margin: 0;
+    opacity: 0%;
```

Comment:
nit: just opacity: 0 ?

Reply:
When you press `command + s`, it will become `0%` like a magic.

Reply:
Our pre-commit hook will chagne `0` to `0%`.